packages/api/src/utils/billing-types.ts

```diff
@@ -132,7 +132,7 @@ export type PaymentSettings = {
 
 export interface UserCustomerService {
   getUserCustomer: (userId: string) => Promise<null|{ id: string }>
-  upsertUserCustomer: (userId: string, customerId: string) => Promise<void>
+  insertUserCustomer: (userId: string, customerId: string) => Promise<void>
```

Comment:
we should capture in the types in kind of 'normal failure' that may happen, e.g. due to a uniqueness constraint, e.g. `UserAlreadyHasCustomer`